javascript/ql/test/library-tests/DOM/event-handler-receiver.js

```diff
@@ -0,0 +1,3 @@
+document.getElementById('my-id').onclick = function() {
+  this.parentNode.innerHTML = '<h2><a href="' + location.href + '">A link</a></h2>'; // NOT OK
```

Comment:
I'm not sure how exploitable this code pattern would be, considering that `location.href` is URL-encoded, but of course that's nothing to do with this PR.

Reply:
I recall there was an issue with IE not encoding the URL when redirected via a `Location` header, so I'm fine with flagging it.

Reply:
And maybe add a comment about not using `getABoundFunctionValue` exactly because bound functions will tend to have a different receiver anyway.